content/about/index.md

```diff
@@ -9,36 +9,44 @@ weight = 10
 
-{{< figure src="./trey_1.jpg" title="Trey Bornmann founded the SVNA in 2016 and then went on to establish the Mountain View Mobile Home Alliance (MVMHA) in 2017." class="float-start w-[200px] me-4 my-0">}}
+Since 2018, we have used our grant money to hold events like a summer barbecue, ice cream socials, Hallowe'en and Thanksgiving events, and to publish newsletters and a website. Last year, we started a popular new tradition -- the Coffee Klatsch, which depending on the weather, is usually held on the first Saturday of each month. 
 
-Trey had been involved in community organizing during Hurricane Katrina when he lived in New Orleans, and he applied his experience working with City government to help Santiago Villa residents challenge those unreasonable rent increases.
+### Annual BBQ
 
-Trey served as President of the SVNA until he and his family moved to Virginia in 2021. His impact on our community is truly hard to overstate, and we're all grateful for his efforts. Thanks, Trey!
+Before Covid, our most popular neighborhood activity was our summer barbecue. We held our first one at the clubhouse in 2018, and it was attended by more than 100 of our residents. Our second one was even more popular. The pandemic put a two-year hold on that annual event, but we hope fire up the grill again in 2025.
 
-Our current President is Bee Hanson, one of the original founders of the SVRA, and steadfast recorder of all SVNA and Mountain View Mobile Home Association (MVMHA) activities. Bee has lived in the neighborhood since 1996, and is looking forward to many more years here.
+### Meetings in the Clubhouse
+
+{{< figure src="./SVNA_pizza_aframe.jpg" class="w-[200px] float-start me-4 my-0">}}
+
+Before the pandemic, we held neighborhood meetings in the clubhouse once a month. This year, some of our grant money went to a Zoom membership, so we can continue to have meetings online no matter what the Coronavirus does next. We'll meet in person soon, but we might also add virtual meetings from time to time.
 
 ## How is SVNA Organized?
 
 The five-member SVNA Board is elected annually. We have an "Election by Acclamation," if the officers run unopposed.
 
-### Our 2022-23 Board
+We'll be having an election for SVNA officers in July. If you like hanging out with a bunch of cool people and doing some good for your neighbors, send email to board@santiagovilla.org.
+
+### SVNA's Board
 
 |                        |               |
 | ---------------------- | ------------- |
 | **President**          | Bee Hanson    |
 | **Vice President**     | Leticia Jones |
 | **Treasurer**          | Randy Wise    |
 | **Communications**     | John Waters   |
-| **Guy Who Does Stuff** | Alex Brown    |
+| **Member At Large**    | Alex Brown    |
```

Comment:
I still prefer my formal title

Reply:
I shoulda waited until I could merge without you knowing

Reply:
And this is why we have branch protection. 😛